eks-worker-al2.json

```diff
@@ -125,14 +125,19 @@
       "pause_before": "90s",
       "remote_folder": "{{ user `remote_folder`}}",
       "inline": [
-        "mkdir -p /tmp/worker/"
+        "mkdir -p /tmp/worker/log-collector-script/"
       ]
     },
     {
       "type": "file",
       "source": "{{template_dir}}/files/",
       "destination": "/tmp/worker/"
     },
+    {
+      "type": "file",
+      "source": "{{template_dir}}/log-collector-script/linux/",
+      "destination": "/tmp/worker/log-collector-script/"
```

Comment:
It might be less confusing to just keep `linux` in the path.

Reply:
I am not sure how customers use the existing path. Hence, I didn't update the existing path to ensure its backward compatible.